test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
```

Comment:
Is there any point in doing this if we then delete all posts?

Reply:
You need a fresh post to run the tests against. As is the cleanup. Otherwise this test could pollute other tests. That said I wonder whether this behavior is baked into the framework itself. Maybe @kevin940726 knows?

Reply:
We don't automatically delete all posts after each test, but we do delete them all once before starting all tests. And yeah, not every test needs a fresh post so we don't do that automatically either. I think putting this in `beforeEach` is a good practice 👍 .